### ISSUE TITLE
Mac Build for CodePad

### ISSUE THREAD

**Issue by Songtech-0912** [thecodepad/codepad#1]
I've created an app bundle for CodePad that runs on Macs. This was built with a 2013 Mac running macOS Catalina 10.15.7. Download the build [here](https://drive.google.com/file/d/1kLYZu3ym0IQydUaSJXmaZm1fCNL5NQGk/view?usp=sharing). Would any developers be willing to add the build to CodePad's releases?

![Screen Shot 2020-12-02 at 5 40 45 PM](https://user-images.githubusercontent.com/61605733/100855873-a5375280-34c5-11eb-8a0a-f2041e44731c.png)

**Comment by apollonian** [comment]
@Songtech-0912 Yes, totally! Can you add a build script with a Darwin target to the package.json too? That'd be wonderful.

**Comment by Songtech-0912** [comment]
@apollonian Thanks! I'll start working on a Darwin build script right away.

**Comment by Songtech-0912** [comment]
@apollonian These build scripts have worked for Darwin and Linux, respectively:

```json
"build:linux": "mkdir dist && electron-packager . CodePad --app-bundle-id=codepad --overwrite --asar=true --platform=linux --icon=build/icons/128x128.png --prune=true --out=dist",
"build:mac": "mkdir dist && electron-packager . CodePad --platform=darwin --arch=x64 --icon=build/icon.icns --prune=true --out=dist",
```

Just add this to the `scripts` section of the `package.json`. Note that the Linux script has an optional dependency on `asar`.

**Comment by apollonian** [comment]
@Songtech-0912 Great! 
Would you be interested in sending a PR? ☺️
